x-pack/plugins/ml/public/application/routing/routes/datavisualizer/datavisualizer.tsx

```diff
@@ -23,6 +21,9 @@ export const selectorRouteFactory = (
 ): MlRoute => ({
   id: 'datavisualizer',
   path: '/datavisualizer',
+  title: i18n.translate('xpack.ml.dataVisualizer.docTitle', {
```

Comment:
Is it possible to set the tab title for the first page in the Data view data visualizer, where you select the data view / saved search? Maybe `Data visualizer | Select data view`?

![image](https://user-images.githubusercontent.com/7405507/151946224-95770e90-2c8c-4b46-a7af-78e9fe9a4ce0.png)

Reply:
Added in https://github.com/elastic/kibana/pull/124125/commits/8ac0b4e349c7f17e079d9add1100fd26f2e33670